giflens/GifLens
# Favorites

Ticket: fixes #20, fixes #26 

## Changes

- Created a Favorites view:
  - click the star in history to add a Gif to favorites
  - one can insert a Gif from the favorites
  - one can remove a Gif from the favorites
- Clicking the item name in one of the views now displays this item in a webview

## How To Test

- build in debug
- reset your history (to get the view from clicking on one element, the click the title command is set in the constructor of the item, old items will not have it)
- do a search to add something to the history
- add this gif to favorites
- insert this gif from the favorites
- click on the item name in both views to check that the gif is displayed
- remove the item from the favorites
